docs/logs/analysis-tab.asciidoc

```diff
@@ -0,0 +1,82 @@
+[role="xpack"]
+[[xpack-logs-analysis-page]]
+== Detecting and inspecting log anomalies
+
+beta::[]
+
+The *Analysis* page in the Logs app allows you to automatically detect some kinds of log anomalies using machine learning.
+
+The analysis automatically highlights periods where the log rate is outside the expected limits and therefore may be anomalous.
```

Comment:
Yep, this is periods of time. 

They are expected limits based on the model defined by the machine learning module, and the "learning" it has done on the datasets to date. Therefore these values will always differ based on the individual dataset. A rate of 10 might be anomalous in one dataset, but not anomalous in another. 

It may be better to use the word "bounds" here over "limit" as that's the ML terminology.